data_architect_misc/data_transformer/configs/LATAM_Argentina/config.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "__comment__": "If we pass None for 'sheet_name' parameter in reading Excel files, pandas will read ALL the sheets.",
+    "__comment__": "The equivalent of None in JSON file is null. That's why we are providing null below to read all the sheets in the file.",
+    "header": 7,
+    "skiprows": 8,
+    "skipfooter": 6,
+
+    "__comment__": "(Optional) Enter the sheet name of the input Excel file to read data from. Default is the first sheet.",
+    "input_sheet_name": "Resumen de Inversiones",
+
+    "__comment__": "(Optional) Enter the sheet name for the output Excel file. Default is 'Sheet1'.",
+    "output_sheet_name": "Sheet1",
+
+    "write_output": true,
+    "output_folder_path": "./output/LATAM_Argentina/",
+    "output_file_name_prefix": "Transformed_Argentina_20200701_20200831_",
+
+    "custom_transform_functions_file": "./transform_functions/latam_argentina_transform_functions.py",
+
+    "functions_to_apply": [
+      {
+        "__function_comment__": "IMPORTANT: Change the folder name to get all files into it",
+        "__function_comment__": "Validate all files to process for this country they must be an amount of files equals to 3",
+        "function_name": "create_new_dataframe_from_input_EXCEL_files",
+        "function_args": ["./input/LATAM_Argentina/August_2020_data"]
+      },
+      {
+        "__function_comment__": "The merged CSV file has unnamed columns. We need to drop them",
+        "function_name": "drop_unnamed_columns"
+      },
+      {
+        "__function_comment__": "In Taiwan data file, there are rows in 'Value' column could have NaN (empty). We need to remove them.",
+        "function_name": "drop_rows_with_matching_string_values",
+        "function_args": [["Mes"],["nan"]]
+      },
+      {
+        "__function_comment__": "Validate the column name if it exist inside the file, if it does not exist this function will create it.",
```

Comment:
Update this function_comment:
In Argentina, we usually receive 3 files. 2 of them contains 'Tema' column, but one of them doesn't. Therefore, we need to add 'Tema' with the function below so that we can use the SAME config file to process all three Argentina files.